test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,36 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
Recall from tutorial whether lumping all possible test conditions inside a Junit test or separating test conditions into their own Junit test is a better implementation.